smoke-tests/SmokeTestGenerator.js

```diff
@@ -68,16 +68,18 @@ function generateScriptBody(isBash, delim) {
 		`${exeName} scanner:run --format junit --target ${buildPath([...projectsPath, 'ts', 'src', 'simpleYetWrong.ts'], delim)} --tsconfig ${buildPath([...projectsPath, 'tsconfig.json'], delim)} --outfile ${buildPath([...resultsPath, 'run2.xml'], delim)}`,
 		`echo "==== Run RetireJS against a folder ===="`,
 		`${exeName} scanner:run --format junit --engine retire-js --target ${buildPath([...projectsPath, 'dep-test-app', 'folder-a'], delim)} --outfile ${buildPath([...resultsPath, 'run3.xml'], delim)}`,
-		`echo "=== Run SFGE against a folder ==="`,
-		`${exeName} scanner:run:dfa --format junit --target ${buildPath([...projectsPath, 'sfge-smoke-app', 'src'], delim)} --projectdir ${buildPath([...projectsPath, 'sfge-smoke-app', 'src'], delim)} --outfile ${buildPath([...resultsPath, 'run4.xml'], delim)}`,
+		`echo "==== Run SFGE's non-DFA rules against a folder ===="`,
```

Comment:
non-DFA GraphEngine rules should be covered by our smoke tests.

Reply:
Instead of SFGE, change to Salesforce Graph Engine (or just Graph Engine) if this text is viewable by outside of our team

Reply:
It's viewable insofar as it's present on our repo. But it's never going to be viewed by an end-user. That said, I'll  change it. Good catch.